ParisBNF/abb/BNFabb94.xml

```diff
@@ -39,23 +39,62 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                                  <material key="parchment">Parchment</material>
                                          </support>
                                       <extent>
-                                          <measure unit="leaf">62</measure>
+                                          <measure unit="leaf" quantity="63">i+61+ii</measure>
                                           <measure unit="leaf" type="blank"/>
-                                          <locus target="#1r #1v #32v #33r #33v #55v #56r #63r #63v"/>
+                                          <locus target="#i #1r #32v #33r #33v #55v #56r #56v #ii"/>
                                           <dimensions type="outer" unit="mm">
                                             <height>210</height>
                                             <width>150</width>
                                           </dimensions>
                                       </extent>
-
-                                      <foliation>European foliation in the upper right corner of the folia.
-                                        </foliation>
-
+                                      <foliation>European foliation in the upper right corner of the folia. The initial unfoliated parchment folium
+                                          is assigned the number <locus target="#i">i</locus>. The last unfoliated parchment folium 
+                                          assigned the number <locus target="#ii">ii</locus>.       
+                                      </foliation>
 <collation>
-    <note>The quire structure cannot be ascertained on
-        the basis of the available images.</note>
-</collation>
-
+    <list>
+        <item xml:id="q1" n="A">
+            <dim unit="leaf">2</dim>
+            <locus from="i" to="1"/>
+        </item>
+        <item xml:id="q2" n="1">
+            <dim unit="leaf">8</dim>
+            <locus from="2r" to="9v"/>
+        </item>
+        <item xml:id="q3" n="2">
+            <dim unit="leaf">8</dim>
+            <locus from="10r" to="17v"/>
+        </item>
+        <item xml:id="q4" n="3">
+            <dim unit="leaf">2</dim>
+            <locus from="18r" to="20v"></locus>
```

Comment:
If the quire has 2 leaves, it should run from 18r to 19v. The folio 20 and 21 is not represented in the quire structure.

Reply:
Thank you! I have corrected it!